### DISCUSSION TITLE
plz help with PyraCanny

### DISCUSSION THREAD

**Discussion by nadora35** [lllyasviel/Fooocus#2672] [Q&A] [answered]
![Untitled](https://github.com/lllyasviel/Fooocus/assets/57017061/70638c9a-0380-4193-8fed-8ea55cd6ef43)

**Comment by mashb1t** [comment, marked as answer]
Please stop spamming this screenshot (4x). Did you modify the code, how did you install Fooocus, did you read the troubleshooting guide, have you sucessfully tried other ControlNets?

**Reply by nadora35** [reply]
First im so sorry for that every time i upload this photo saying fiald. So i try to upload again. I tried to modify the come but failed to. So i deleted all controlnets and downloaded it ant its works thank you very much for your attention
